rsyslog/Sanity/elasticsearch/elasticsearch.fmf

```diff
@@ -0,0 +1,71 @@
+# run e.g. with:
+#   tmt run -a -vvvddd provision --how minute -i rhel-8 report --verbose plans --name /rsyslog/Sanity/elasticsearch/elasticsearch/v7.3.0
+
+context:
+    EStest: 1
+
+discover:
+  - how: fmf
+    test:
+      - /rsyslog/Sanity/elasticsearch/test
+execute:
+  - how: tmt
+
+/v1.3.1:
+    prepare:
+      - how: shell
+        script:
+          - rpm -q elasticsearch || yum install -y ./rsyslog/Sanity/elasticsearch/resources/elasticsearch-1.3.1.noarch.rpm
+          - systemctl daemon-reload
+          - sed -i 's/#ES_JAVA_OPTS=/ES_JAVA_OPTS=\"-Xss1024k\"/' /etc/sysconfig/elasticsearch
+          - cd /etc/yum.repos.d/; wget http://nest.test.redhat.com/mnt/qa/scratch/dapospis/yum.repos.d/scratch/alakatos/rsyslog/8.2102.0-15.el8/rsyslog-8.2102.0-15.el8-scratch.repo
+    finish:
+      - how: shell
+        script:
+          - yum remove -y elasticsearch
+/v7.3.0:
+    prepare:
+      - how: shell
+        script:
+          - update-crypto-policies --set LEGACY
+          - rpm -q elasticsearch || dnf install -y https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.3.0-x86_64.rpm
+          - cd /etc/yum.repos.d/; wget http://nest.test.redhat.com/mnt/qa/scratch/dapospis/yum.repos.d/scratch/alakatos/rsyslog/8.2102.0-15.el8/rsyslog-8.2102.0-15.el8-scratch.repo
```

Comment:
I can see this line used in every single ES version, I believe at this point the elasticsearch package should already be installed. Does it have any special meaning or it's a leftover?